frontend/lang/compiled/zu.json

```diff
@@ -1 +1,160 @@
-{}
```

Comment:
You're right: https://github.com/Vizzuality/heco-invest/tree/develop/frontend#pipeline. Maybe we could update `.gitignore` to avoid this in the future? I know I also pushed the `compiled` folder unintentionally in the past.